components/ws-manager/pkg/manager/monitor.go

```diff
@@ -498,6 +498,11 @@ func actOnPodEvent(ctx context.Context, m actingManager, manager *Manager, statu
 					}
 				}
 
+				// delete PVC because the workspace pod is never ready
+				if err = manager.deleteWorkspacePVC(ctx, pod.Name); err != nil {
+					log.Error(err)
```

Comment:
can you change this back to WithError and provide a short blurb that error is due to failure to delete workspace pvc? otherwise it will be hard to understand where error might be getting logged and coming from.

Reply:
Because if the error happens, the error message from the `deleteWorkspacePVC` would be `return xerrors.Errorf("cannot delete workspace pvc: %w", err)`. And I think it contains enough information. WDYT?